examples/01a-Getting-started.ipynb

```diff
@@ -0,0 +1,632 @@
+{
```

Comment:
<p>We should summarize the end of the notebook with something like:</p><p> </p><pre class="ql-syntax" spellcheck="false">## Conclusion
We demonstrated how to train a TensorFlow Keras model with Merlin dataloader. Merlin dataloader can accelerate existing TensorFlow pipelines with minimal code changes. 

## Next Steps
Merlin dataloader is part of NVIDIA Merlin, a open source framework for recommender systems. In this example, we looked only on a specific use-case to accelerate existing training pipelines. We provide more libraries to make recommender system pipelines easier:
- NVTabular is a library to accelerate and scale feature engineering
- Merlin Models is a library with high-quality implementations of popular recommender systems architectures 

The libraries are designed to work closely togethes. We recommend to checkout our exmaples:
- Getting Started with NVTabular: Process Tabular Data On GPU
- Getting Started with Merlin Models: Develop a Model for MovieLens 

In the example, From ETL to Training RecSys models - NVTabular and Merlin Models integrated example, we explain how the close collaboration works.
</pre><p> </p><p>Can you add links for the libraries and examples?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/dataloader/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>absolutely, done! 🙂🙌</p>